sdks/python/apache_beam/runners/worker/sideinputs.py

```diff
@@ -73,7 +73,24 @@ def __init__(self,
     # Whether an error was encountered in any source reader.
     self.has_errored = False
 
-    self.read_counter = read_counter or opcounters.TransformIOCounter()
+    self.read_counter = read_counter or opcounters.NoOpTransformIOCounter()
+    experiments = RuntimeValueProvider.get_value('experiments', list, [])
+    if 'sideinput_io_metrics' in experiments:
```

Comment:
When are switching to versioned experiments?

Reply:
Hi Ahmet. This experiment was already ongoing when we decided to start using versioned experiment strings. Since we have only rolled out to internal projects, it must be fine to keep the first string version (and later use `_v2`). WDYT?

Reply:
Sounds good. I just want to avoid a future auto-enabling.